gneiss/balances.py

```diff
@@ -34,9 +34,8 @@
 try:
     from ete3 import Tree, TreeStyle
     from gneiss.layouts import default_layout
-except ImportError:
-    raise ImportWarning('ete3 is not installed.  '
-                        'ETE3 style visualizations will not be available.')
+except:
```

Comment:
This will still print an error if the function is called. 

Apparently `raise ImportWarning` raises an exception - which breaks the entire CLI interface.
Another options is to use `warnings.warn(...)`.  However, this isn't what we want, since this warning will be printed every time a command line call is made.